docs/community/model-providers/mlx.md

```diff
@@ -0,0 +1,238 @@
+# MLX (Apple Silicon)
+
+{{ community_contribution_banner }}
+
+!!! info "Language Support"
+    This provider is only supported in Python.
+
+!!! warning "Platform Requirement"
+    MLX is designed specifically for Apple Silicon (M1/M2/M3/M4 chips) and CUDA environments. Requires Python ≤3.13 on macOS or Linux.
+
+[MLX](https://github.com/ml-explore/mlx) is Apple's machine learning framework optimized for Apple Silicon. The `strands-mlx` provider enables local inference, LoRA fine-tuning, and vision capabilities on Apple Silicon devices with support for mlx-community quantized models.
+
+## Installation
+
+To use MLX models with Strands Agents on Apple Silicon:
+
+```bash
+# Create virtual environment with Python 3.13
+uv venv --python 3.13 && source .venv/bin/activate
+
+# Install dependencies
+uv pip install strands-mlx strands-agents-tools
+```
+
+Or with pip:
+
+```bash
+pip install strands-mlx strands-agents-tools
+```
+
+## Quick Start
```

Comment:
Thoughts about relabeling this to `## Usage` to match the other providers?

Reply:
Strands-MLX repository have more documentation down below, trying to match with other providers does not makes sense to me. :/